### ISSUE TITLE
Add UUID component

### ISSUE THREAD

**Issue by peter279k** [alphazframework/framework#219]
**Is your feature request related to a problem? Please describe.**

I'm not sure we have to provide this component by dafault on our Zest framework.

**Describe the solution you'd like**

We can look at the [ramsey/uuid](https://github.com/ramsey/uuid) to know more details about `UUID`.

And this official [uniqid function reference](https://www.php.net/manual/en/function.uniqid.php) can also let us know `UUID`.

**Comment by lablnet** [comment]
Yes i was thinking before, its should be part of Zest Framework, its good if you could send `PR`
what do you think @peter279k

**Comment by peter279k** [comment]
Thanks @Lablnet, and I also have the similar issue on [this](https://github.com/selective-php/util/issues/3) :).

**Comment by lablnet** [comment]
I have following approach in my mind

1. Generate random characters 10 or 15 by using `salts` method from `Site` class.
2.  Generating random number from `100000` to `1000000` and then multiply by it `time()`.
3. Then concatenate those and shuffle it.
4. For providing additional sticky rule we can make optional feature allow user to validate it, like when ever it generate UUID we can append it to json, and verify it either already exist or not if so, do it again.... but this will be optional 

What do you think @peter279k

**Comment by peter279k** [comment]
@Lablnet, thanks for your reply.

The UUID means the unique ID value.

It's also a random value, this generation has the standard defined by [RFC-4122](https://tools.ietf.org/html/rfc4122).

It's also have the implementation [here](https://packagist.org/packages/ramsey/uuid).
